plenum/test/view_change/test_no_propagate_request_on_different_last_ordered_before_vc.py

```diff
@@ -52,11 +52,11 @@ def test_no_propagate_request_on_different_last_ordered_on_backup_before_vc(loop
     # Backup primary replica set new_view and seq_no == 1, because of primary batch
     looper.run(eventually(check_last_ordered, non_primaries,
                           slow_instance,
-                          (last_view_no + 1, 1)))
+                          (last_view_no + 1, 2)))
```

Comment:
Why is it `2` here?

Reply:
1 - batch with primaries after view_change and 1 request will be ordered from previous view, because it was not ordered.